File: Lib/Telerik/Examples/QuickStart/Docking/MDI/OrderForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace Telerik.Examples.WinControls.Docking.MDI
{
    public partial class OrderForm : Form
    {
        public OrderForm()
        {
            InitializeComponent();
        }

        private void ordersBindingNavigatorSaveItem_Click(object sender, EventArgs e)
        {
            this.Validate();
            this.ordersBindingSource.EndEdit();
            this.ordersTableAdapter.Update(this.nwindRadGridView.Orders);

        }


        protected override void OnLoad(EventArgs e)
        {
            base.OnLoad(e);

            // TODO: This line of code loads data into the 'nwindRadGridView.Orders' table. You can move, or remove it, as needed.
            this.ordersTableAdapter.Fill(this.nwindRadGridView.Orders);
        }
    }
}

File: Lib/Telerik/Examples/QuickStart/Docking/MDI/OrderForm.vb
Imports System.ComponentModel
Imports System.Text

Namespace Telerik.Examples.WinControls.Docking.MDI
	Partial Public Class OrderForm
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub ordersBindingNavigatorSaveItem_Click(ByVal sender As Object, ByVal e As EventArgs)
			Me.Validate()
			Me.ordersBindingSource.EndEdit()
			Me.ordersTableAdapter.Update(Me.nwindRadGridView.Orders)

		End Sub


		Protected Overrides Sub OnLoad(ByVal e As EventArgs)
			MyBase.OnLoad(e)

			' TODO: This line of code loads data into the 'nwindRadGridView.Orders' table. You can move, or remove it, as needed.
			Me.ordersTableAdapter.Fill(Me.nwindRadGridView.Orders)
		End Sub
	End Class
End Namespace